account-kit/rn-signer/ios/implementation/KeychainHelper.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import Security
+import CryptoKit
+
```

Comment:
this is a nit but for the constants lets make them upper case so it is obvious in the code i.e.

`let ACCOUNT_KIT_EPHEMERAL_PRIVATE_KEY = "accountKitEphemeralPrivateKey"`